src/utils/index.ts

```diff
@@ -11,3 +11,8 @@ export async function wait<T = unknown>(time: number, value?: T): Promise<T | un
         setTimeout(() => resolve(value), time);
     });
 }
+
+export function normalizePathSlashes(path: string) {
```

Comment:
i think this function actually deserves tests

Reply:
Added